src/compareImage.test.js

```diff
@@ -62,15 +62,31 @@ describe('Compare Image', () => {
   it('returns correct value if only difference above threshold', async () => {
     Jimp.diff.mockReturnValue({ percent: 0.02 });
 
-    const result = await compareImage(mockConfig, 'test');
-    expect(result).toEqual('no mismatch found ✅');
+    try {
+      await compareImage(mockConfig, 'test');
+    } catch (err) {
+      expect(err.message).toEqual(`mismatch found❗
+    Result:
+      distance: 0
+      diff: 0.02
+      misMatchThreshold: 0.01
+  `);
+    }
   });
 
   it('returns correct value if only distance above threshold', async () => {
```

Comment:
add `expect.assertions(1);` here to make sure the `expect` in the `catch` is hit